File: src/client/java/io/github/seggan/deathview/client/mixin/DeathScreenAccessor.java
package io.github.seggan.deathview.client.mixin;

import net.minecraft.client.gui.screens.DeathScreen;
import org.spongepowered.asm.mixin.Mixin;
import org.spongepowered.asm.mixin.gen.Accessor;

@Mixin(DeathScreen.class)
public interface DeathScreenAccessor {

    @Accessor("delayTicker")
    int getTicksSinceDeath();
}

File: src/client/kotlin/io/github/seggan/deathview/client/mixin/DeathScreenAccessor.kt
package io.github.seggan.deathview.client.mixin

import net.minecraft.client.gui.screens.DeathScreen
import org.spongepowered.asm.mixin.Mixin
import org.spongepowered.asm.mixin.gen.Accessor

@Mixin(DeathScreen::class)
interface DeathScreenAccessor {

    @get:Accessor("delayTicker")
    val ticksSinceDeath: Int
}
